hkmc2/shared/src/main/scala/hkmc2/codegen/js/JSBuilder.scala

```diff
@@ -179,6 +184,10 @@ class JSBuilder(using Elaborator.State, Elaborator.Ctx) extends CodeBuilder:
                     doc" # ${td.sym.nme}($params) { #{  # ${
                       bodyDoc
                     } #}  # }"
+                  case td @ FunDefn(_, Nil, bod) =>
+                    doc" # ${td.sym.nme}() { #{  # ${
```

Comment:
Argh, I now just realized that in fact, for classes, getters don't need any special treatment, since JS already supports them! They should be of the form `get ${td.sym.nme}() { ... }`. Then there is nothing special to do for them in this context and we only need to take care of those getters that are NOT in object/class scopes. There is no need for the module restriction.

Note: IIRC you can check if something is or isn't in a class/object scope by checking whether the symbol has an owner.

Reply:
I've fixed all related implementations.